qiime2/sdk/tests/test_usage.py

```diff
@@ -149,19 +149,29 @@ def test_use_merge_feature_table(self):
         use = usage.DiagnosticUsage()
         action.examples['variadic_input_simple'](use)
 
-        self.assertEqual(len(use.recorder), 3)
+        self.assertEqual(len(use.recorder), 7)
```

Comment:
Posting here because its the closest I can get to the line of interest, but can you rename this test, its a copy-paste error from ye olde days.